mozilla/foundation.mozilla.org
# properly hook up the callpower CTA social share buttons

Closes https://github.com/mozilla/foundation.mozilla.org/issues/7690 

Testing:

- create a "callpower" snippe with the following SP values, and a campaign id that you ask @pomax about in slack:

> ![image](https://user-images.githubusercontent.com/177243/139740563-9bb6bc05-e2d5-40c3-a584-df570445c734.png)

- create a new campaign and set its CTA to this new snippet (it'll be the last one on the last page for the cta snippet chooser).

- view the campaign page and verify that it shows the CTA with phone number and zipcode fields, and that filling those in and calling leads to:
  - a call to your phone
  - a "thank you" view change with social buttons, which
    - generate prepulated facebook/twitter/email shares
    - copy the page link to your clipboard, for the "copy" button.
